src/plugins/intel_cpu/src/nodes/dft.cpp

```diff
@@ -16,12 +23,348 @@
 #include <ngraph/opsets/opset7.hpp>
 
 using namespace dnnl;
+using namespace dnnl::impl;
+using namespace dnnl::impl::utils;
+using namespace dnnl::impl::cpu::x64;
 using namespace InferenceEngine;
+using namespace Xbyak;
+
+#define GET_OFF_DFT(field) offsetof(jit_args_dft, field)
```

Comment:
Please move all JIT related code into the _intel_cpu/src/nodes/kernels_ folder .

Reply:
Done